client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -667,6 +674,8 @@ export default connect(
 			isPlansListFetching: isRequestingPlans( state ),
 			isSitePlansListFetching: isRequestingSitePlans( state, selectedSiteId ),
 			planSlug: getUpgradePlanSlugFromPath( state, selectedSiteId, props.product ),
+			isJetpackNotAtomic:
```

Comment:
Oh wow, this really needs a dedicated selector.

Reply:
Agreed, the situation is not very good right now and could do with a complete rethinking. Not in this PR though 😉